lib/wprof/conf_wprof.rb

```diff
@@ -0,0 +1,25 @@
+module WProf
+  class Config
+    DEFAULTS_CONFIGS = {
+      db_runtime: true,
+      reporter_type: 'LOGGER',
+      csv_type: 'SPLIT',
+      async: false,
+      httparty_methods_to_trace: %w[get put delete post patch].freeze,
+      external_url: nil,
+      external_headers: nil,
+      custom_methods: nil,
+      file_path: Rails.root.to_s
+    }.freeze
+
+    class << self
```

Comment:
If we only have one class method we could defined it as:
```
def self.get_value(param)
 . . . 
end
```

Instead of prepending the `self` in all the methods definition. But it's not a big deal

Reply:
The all class must be have class methods (the most part at less), so, if later we add news methods the job it's done. I want leave this.

Reply:
But in the other coment you said:
> I though that, but is the only 'support' method and it only is used there, i will leave it for now... if later we puts news supports functions, ok, I will move on them into a PORO or helper.

You must make a convention decision. Are we going to be reactive or proactive in your code?

If you are convinced that you are going to add new methods to this class then leave de self class call in all the methods of this class. If you do that i would suggest you to move `_format_time` to a PORO so we are consistent, have a cleaner code and we follow the SRP on OO.

Or you could reactive to future change and remove the `<< self` call and replace it with `self.<method>`

Am i being a little hard in this decision? Yeah maybe because I think it's easy to make this kind of decision now and not have inconsitant code.

I'm happy which wherever convention you chose in your code because it's a great gem but I will like to be consistent on the code style.

Reply:
Ok, i will accept the ambiguous my words was and also I'm agreed with try to maintenance the SRP. I changed it, probably i would have changed it later so, I doing it now.